### ISSUE TITLE
Improved usage example

### ISSUE THREAD

**Issue by wernight** [Grafikart/Docker-ngrok#1]
Could have some more elaborated example like https://github.com/fnichol/docker-ngrok but also including an [API call](https://ngrok.com/docs#client-api) to programmatically retrieve the assigned domain.

PS: Nice version, I wonder if it'd not be better based off the default busybox?

**Comment by Grafikart** [comment]
Yeah getting the assigned domain is a real problem, I asked ngrok directly if they could include this in the log, that would be the best solution.

For the default busybox you may be right, i'll try to see if it work and i'll make the change ^^
